RFC-004-revising-licenses-page.md

```diff
@@ -0,0 +1,118 @@
+# RFC004 - Revising website licenses terms/page
+
+## Proposed by
+
+Ryan Macklin ([@macklin](https://thegooddocs.slack.com/team/U01DYRWG43X))
+Initially submitted on 2021-07-13
+
+## Current status
+
+- [ ] Draft
+- [x] Under discussion (until 2021-07-27, assuming two weeks post initial submission)
+- [ ] Final comment and voting (until YYYY-MM-DD) {{Add date after selecting this status.}}
+- [ ] Accepted
+- [ ] Rejected
+- [ ] Implemented
+- [ ] Deferred
+- [ ] Withdrawn
+
+## Proposal overview
+
+I believe we should rethink the licensing terms for website content, as listed on [thegooddocsproject.dev/licenses/](https://thegooddocsproject.dev/licenses/). As they're currently written, they provision everything as [Creative Commons Attribution 4.0](https://creativecommons.org/licenses/by/4.0/) (a.ka. "CC BY 4.0), including all text, images, HTML code, and anything else.
+
+We may not always have the rights to relicense what we put on our page, and not every contributor is comfortable with having their info and likeness released as publicly usable.
+
+### Scope
+
+The scope of this is about the website's content. Licensing for our products (templates, doc tools, etc.) aren't part of this discussion, though this proposal does indicate where the primary home for those licenses should be.
+
+## Motivation
+
+For the upcoming blog, not everything we release will be ours. If we use stock photos for the purposes of social media optimization, we won't have the rights to relicense that content. We also may not have the rights to relicense other content we'll reuse, including any HTML/CSS/scripting to generate the pages.
+
+On a personal level, I don't release my likeness to public use, and the current website license terms makes me wary of continuing with the project. I used to work in games (and still do at times), and I've had too many death threats (honestly, one such threat is too many) and have seen too many friends be impersonated to want to give bad actions a contractual right to use my likeness however they like. (Bad actors will still use likenesses, but at least they can't defend their actions by citing a license that allows them to use one's info.)
+
+### Blocking
+
+[RFC002 (Procope/blog & video)](https://github.com/thegooddocsproject/request-for-comment/pull/3) is blocked by this proposal.
+
+## Proposal
+
+I'd like to shift the burden of licensing to the individual elements, rather than have a pure blanket. That means:
+
+- Our products (templates, doc tools, etc) have the BSD0 license in their readme or license text files, and not rely on the license website page for that notice
```

Comment:
praise: I'm supportive of licenses being applied on a per-repository basis, in line with standard open source practices. We can then revise wording in thegooddocsproject.dev/licenses/ to be a summary of license terms within our repositories, where {repository}/LICENSE.txt is referred to as the point of truth.
This can be framed as:
"As defined in each repository's LICENSE.txt file, ... {provide summary}"
thought: I wouldn't want to introduce any special case which deviate from the repository license file as I feel this is going to become too complicated to manage and consequently will be broken (due to ignorance. indifference, laziness, annoyance or possibly even spite due to frustration due to the extra red tape required).

Reply:
100%. Though that's a decision to be handled by the template sub-orgs, it seems like we'd have an interal (don't pardon the pseudo-pun) *template* for our license file. That could be nearly all the same text for the license, perhaps with particular license acknowledgments at the end as needed. (Again, for the template sub-orgs to figure out.)

Reply:
Thanks for the clarification. Happy to close this out.